src/views/movie_view.js

```diff
@@ -0,0 +1,30 @@
+import Backbone from 'backbone';
+import Movie from '../models/movie';
+import MovieList from '../collections/movie_list';
+
+const MovieView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    this.collection = params.collection;
+  },
+
+  render() {
+    const compiledTemplate = this.template(this.model.toJSON());
+    this.$el.html(compiledTemplate);
+    return this;
+  },
+
+  events: {
+    'click': 'showMovie',
+  },
+
+  showMovie(event){
+    let found = this.collection.where({overview: this.model.get('overview')}).length > 0;
+    this.model.set('inInventory', found);
+
+    this.bus.trigger('addToCollection', this.model);
```

Comment:
I'm not sure `addToCollection` is a good name for this event, since you haven't added anything to a collection. Perhaps something like `selectMovie` would be more mnemonic.